linux_os/guide/system/permissions/files/permissions_within_important_dirs/dir_permissions_library_dirs/rule.yml

```
@@ -0,0 +1,53 @@
+documentation_complete: true
+
+title: |-
+    Verify that Shared Library Directories Have Restrictive Permissions
+
+description: |-
+    System-wide shared library directories, which contain are linked to executables
+    during process load time or run time, are stored in the following directories
+    by default:
+    <pre>/lib
+    /lib64
+    /usr/lib
+    /usr/lib64
+    </pre>
+    Kernel modules, which can be added to the kernel during runtime, are
+    stored in <tt>/lib/modules</tt>. All sub-directories in these directories
+    should not be group-writable or world-writable. If any file in these
+    directories is found to be group-writable or world-writable, correct
+    its permission with the following command:
+    <pre>$ sudo chmod go-w <i>DIR</i></pre>
+
+rationale: |-
+    If the operating system were to allow any user to make changes to software libraries,
+    then those changes might be implemented without undergoing the appropriate testing
+    and approvals that are part of a robust change management process.
+
+    This requirement applies to operating systems with software libraries that are accessible
+    and configurable, as in the case of interpreted languages. Software libraries also include
+    privileged programs which execute with escalated privileges. Only qualified and authorized
+    individuals must be allowed to obtain access to information system components for purposes
+    of initiating changes, including upgrades and modifications.
+
+identifiers:
+    cce@sle15: CCE-85753-2
+
+severity: medium
+
+references:
+    cci@sle15: CCI-001499
```

```suggestion
    disa: CCI-001499
```

Unfortunately the correct reference here is `disa`, see in the references table here: https://complianceascode.readthedocs.io/en/latest/manual/developer/06_contributing_with_content.html#rules